src/cli/playpass-client.ts

```diff
@@ -55,7 +68,10 @@ export default class PlaypassClient {
                 "X-API-TOKEN": this.authToken
             }
         })
-            .then((a: AxiosResponse<{items: Game[]}>) => a.data.items);
+            .then((a: AxiosResponse<Games>) => {
+                PlaypassClient.validateResponse(a, "Failed to retrieve games.");
+                return a.data.items;
+            });
```

Comment:
Have you been able to test this? I think the server responds with 4xx status codes with an error, and axios is set up to immediately reject the promise on a non-200 status code, before it would get to this code path. We either need to change the server to always return 200 or have the client handle non-200.

Reply:
This is backwards compatible for the moment. I want to change the API to return a result and leave error status codes for actual errors.